tasks/main.yml

```diff
@@ -9,3 +9,8 @@
   vars:
     ansible_python_interpreter: "{{ os_images_venv ~ '/bin/python' if os_images_venv != None else old_ansible_python_interpreter }}"
   when: os_images_upload | bool
+
+- import_tasks: cleanup.yml
+  vars:
+    ansible_python_interpreter: "{{ os_images_venv ~ '/bin/python' if os_images_venv != None else old_ansible_python_interpreter }}"
+  when: os_images_cleanup | bool
```

Comment:
could just check for os_images_delete_list  being non empty?

Reply:
This is a precuation, I don't want it to work with just providing the list.

Reply:
This is a precuation, I don't want it to work with just providing the list.

Reply:
Isn't that going to confuse people?

Reply:
In case a list is automatically generated, I would want to have an option to turn it off via variable.

Reply:
Here's an example configuration: https://github.com/stackhpc/smslab-config/pull/94